server/gokbg3/grails-app/services/gokbg3/RestMappingService.groovy

```diff
@@ -151,8 +152,24 @@ class RestMappingService {
         }
         else {
           switch (p.type) {
+            case Float.class:
+              if (p.name == 'price') {
+                String pstring = obj[p.name] ? "${obj[p.name].trunc(2)}" : null
+
+                if (pstring) {
+                  if (!pstring.contains('.')) {
```

Comment:
Are we sure that pstring does not contain other separator chars, esp. `,` ?

Reply:
I would think so, `Float.toString()` should always be of the form `123.23345`